engine/src/flutter/shell/platform/android/platform_view_android_jni_impl.cc

```diff
@@ -488,6 +488,8 @@ static void DispatchSemanticsAction(JNIEnv* env,
                                     jint action,
                                     jobject args,
                                     jint args_position) {
+  // TODO(team-android): Remove implicit view assumption.
+  // https://github.com/flutter/flutter/issues/142845
```

Comment:
Looks like this comment should be moved to `PlatformViewAndroid::DispatchSemanticsAction`